### ISSUE TITLE
[Bug] Mana Regen stops if the Total amount of mana is below 100

### ISSUE THREAD

**Issue by Simplfnk** [iron431/Irons-Spells-n-Spellbooks#425]
### Observed behaviour
Manag reagen calculates the total amount of mana and for every 100 points of mana it regenerates 1 point, since my item dropped the mana below 100 ( 100 - 33%  =  66 points )  the mana regen becomes 0
when equipping this Curio item the mana does not regenerate it stops completely until i unequip the item it starts to regenerate . 

the problem with which mana regen is calculated, you don't need any item that boosts mana regen just focus on increasing your Mana, and that will directly increase your mana regen because for every extra 100 points of Mana, you get 100%  increase in mana Regen, so I don't know if this is a bug or a feature

```ruby
if (CuriosApi.getItemStackSlots(stack , slotContext.entity()).containsKey(slotContext.identifier()) && isEnabled && TogElMaxMana.get()) {
			modifiers.put(AttributeRegistry.MAX_MANA.get() ,
					new AttributeModifier(MAX_MANA_UUID , "" , -0.33f ,
							AttributeModifier.Operation.MULTIPLY_TOTAL));
		}
```		 

### Expected behaviour

my total mana should be -33% but still regenerate 

### Steps to reproduce

use  CurioItem with this code
```ruby 
if (CuriosApi.getItemStackSlots(stack , slotContext.entity()).containsKey(slotContext.identifier()) && isEnabled && TogElMaxMana.get()) {
			modifiers.put(AttributeRegistry.MAX_MANA.get() ,
					new AttributeModifier(MAX_MANA_UUID , "" , -0.33f ,
							AttributeModifier.Operation.MULTIPLY_TOTAL));
		}
```
### Server Type

Single Player

### Iron's Spells N Spellbooks version

1.20.1-3.1.4

### Forge version

120.1

**Comment by iron431** [comment]
will be fixed in the next update

**Comment by iron431** [comment]
fixed in 3.1.5
